README.md

```diff
@@ -0,0 +1,53 @@
+# WUDU*
+
+Application that lets you create, save, visualise, analyse and modify 4 different types of graphs (see *) 
+
+## Features
+
+- Support 4 types of graphs (see *)
+- Use all in all - 9 algorithms
+- Store graphs anywhere you want: SQLite, Neo4J, JSON
+- Drag, zoom, replace nodes
+
+![mainscreen_screenshot](https://github.com/spbu-coding-2023/graphs-graph-2/assets/64466788/9c708a11-dc6e-4cf9-a848-44f21dec7a37)
+
+## Usage
+
+Firstly - clone the last release/version of our app. Since our app uses Gradle build system - you can use these commands:
```

Comment:
I apologise, we haven't mentioned( While developing our app, we used 21 JDK. I mistakenly thought that gradle wrapper will use the specified JDK, but then i realised, that we didn't specify it in the project ;)

Maybe it can run on previous versions, but we haven't test. It cant be run at JDK < 21